CONTENT/about.markdown

```diff
@@ -1,49 +1,79 @@
 [//]: # (@TODO - make this file nice)
 
-# Manifesto
-(Philosophy/Values/Goals)
-=> Link to Github
```

Comment:
The `=> link to github` was a hint that here might be the appropriate place to link to our `visionandvalues.markdown` and maybe our `wallofinspiration.markdown`.
I dont know where else to include them.
Any thoughts?